regression/storage.py

```diff
@@ -52,7 +52,7 @@ def setUp(self):
     def tearDown(self):
         with storage.Batch() as batch:
             for bucket_name in self.case_buckets_to_delete:
-                storage.Bucket(bucket_name, connection=batch).delete()
+                storage.Bucket(bucket_name).delete(connection=batch)
```

Comment:
Should this be necessary? Will it not come in as `top` in `_require_connection`?